Physicist/Physicist/Actor/Player.cs

```diff
@@ -255,7 +284,7 @@ private Vector2 GetMovementSpeed(KeyboardDebouncer state)
 
         private void CreateButtons()
         {
-            this.Body.CollisionCategories = PhysicistCategory.Player;
+            this.Body.CollisionCategories = PhysicistCategory.Player1;
```

Comment:
We need another solution for this if we're thinking about using multiple players. Namely, this needs to be set dynamically somewhere. Food for thought